patch/llpcPatchBufferOp.cpp

```diff
@@ -1595,192 +1595,159 @@ Value* PatchBufferOp::ReplaceLoad(
     }
 
     SmallVector<Value*, 8> partLoads;
+    Type* smallestType;
```

Comment:
smallestType -> pSmallestType = nullptr
Initialize all local variables.

Reply:
I don't think the pPrefix review comments are helpful. We agreed that we would go away from this style.

Reply:
Yes, we are OK without "p" prefix. Currently, as discussed, recently added codes follow the style of surrounding codes. This is just to keep coding consistency, not making new commers feel confusing which style to follow in a mixed environment.
I think Tim will start style transition soon. It will not be a problem in the near future.